README

```diff
@@ -225,7 +225,7 @@ REQUIREMENTS:
         polkit (optional)
 
         To build in directory build/:
-          meson setup build/ && meson compile -C build/
+          meson setup build/ && ninja -C build/
```

Comment:
interesting, i though meson prefered the syntax via meson, because they wanted to permit some level of abstraction of used backend?

Reply:
That's a matter of some opinion.

Many ninja targets are simply not allowed to be run via `meson compile` because they don't compile anything. Not all versions of Meson actually support meson compile (some versions simply try to configure a new build directory at ./compile/ instead) and in the best case scenario, it will first check whether you are using ninja or Microsoft Visual Studio, and effectively run `ninja "$@"` or `msbuild "$@"`.